tests/infra/__init__.py

```diff
@@ -15,7 +16,15 @@ def setUp(self):
 @contextlib.contextmanager
 def encoded_bytes_stream():
     old_stdout = sys.stdout
-    sys.stdout = TextIOWrapper(BytesIO(), sys.stdout.encoding)
+    sys.stdout = io.TextIOWrapper(io.BytesIO(), sys.stdout.encoding)
     yield
     sys.stdout.close()
     sys.stdout = old_stdout
+
+def upload_data(uri: str, data: bytes):
+    if uri.startswith("gs://"):
+        bucket, key = uri[5:].split("/", 1)
+        Client().bucket(bucket).blob(key).upload_from_file(io.BytesIO(data))
+    else:
```

Comment:
Same as above, though possibly checking for `drs://` if this is never expected to run `drs` (and maybe a comment about why no drs here).

Reply:
This is not a user space function, but I'll add a "not supprted".

Devs attempting to upload to drs uris have deep misconceptions.

Reply:
On second thought, I'm going to leave this as-is. Adding arbitrary checks for hypothetical developer misconceptions doesn't seem worth the complexity.